package.json

```diff
@@ -89,6 +91,7 @@
   "scripts": {
     "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/ script/ || exit 0",
     "postlint": "node ./script/npm_lint_plugins.js",
+    "lint:local": "./node_modules/.bin/eslint -c .eslintrc webpack/ script/ || exit 0",
```

Comment:
whats the difference to lint task?

Reply:
This is just something I've been using locally to lint only foreman (skips the `postlint` script). I have some questions about the `postlint` process explained [here](https://github.com/theforeman/foreman/pull/4946#issuecomment-343533540) but I'm happy to remove this if it's not helpful to others.